e2e/nomostest/ntopts/multi_repo.go

```diff
@@ -104,3 +108,10 @@ func WithReconcileTimeout(timeout time.Duration) func(opt *New) {
 		opt.ReconcileTimeout = &timeoutCopy
 	}
 }
+
+// RepoSyncPolicy specifies PolicyRule(s) to grant NS reconcilers
+func RepoSyncPolicy(policy ...rbacv1.PolicyRule) Opt {
```

Comment:
How about just RepoSyncPermissions to make it consistent with the struct field?

Reply:
Done